keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -311,6 +314,32 @@ These cases will be added in the existing e2e tests:
 - [X] Allowing time for feedback (1 year).
 - [X] Risks have been addressed.
 
+### Graduation Criteria of Options
+
+In 1.26 we are releasing this feature to Alpha. We propose the following management of TopologyManager policy options graduation:
+- `TopologyManagerPolicyAlphaOptions` is not enabled by default. Topology Manager alpha options (only one as of 1.26), are hidden by default
+ and only when `TopologyManagerPolicyAlphaOptions` feature gate is enabled user is able to use alpha options.
+  `TopologyManagerPolicyAlphaOptions` will not exist if there are no options in alpha stage.
+- `TopologyManagerPolicyBetaOptions` is not enabled by default. Topology Manager beta options (none as of 1.26), are hidden by default
+ and only when `TopologyManagerPolicyBetaOptions` feature gate is enabled user is able to use beta options.
+`TopologyManagerPolicyBetaOptions` will not exist if there are no options in beta stage (true for 1.26 release).
+- `TopologyManagerPolicyOptions` is enabled by default. Topology Manager stable options (none as of 1.26), are visible by default
```

Comment:
Since this is an alpha feature, `TopologyManagerPolicyOptions` should be disabled by default.

Reply:
Yes. This needs to be updated.

Reply:
Fixed.